### DISCUSSION TITLE
Distribution of scan result to multiple api keys

### DISCUSSION THREAD

**Discussion by samogod** [projectdiscovery/uncover#473] [Ideas]
There should be a feature that the multiple api keys given in the provider config will get the whole result for the given input. In the current situation, it randomly selects an api key and finishes without getting the whole result. My other api keys are not even touched. Such a feature would be really good. 

Thanks.

**Comment by GeorginaReeder** [comment]
Hey @samogod ! Thanks for your idea - could you please go ahead and create a feature request for this? We'll be happy to take a look from there :)
